sql/moz-fx-data-marketing-prod/ga_derived/mobile_installs_adjust_v1/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+description: Aggregated metrics from Adjust data
+friendly_name: Adjust mobile installs
+labels:
+  incremental: true
```

Comment:
if it's incremental the date filter should have a macro

Reply:
@sean-rose and @bani, if I turn this false, should I have an init.sql to create the base table?

Reply:
The label is just informational, it doesn't control anything.  I think Vanessa's point was that the label should accurately reflect whether the query is incremental or not.

Regarding having an `init.sql`, with the query currently not being incremental there's no need for an `init.sql`.  If you change the query to be incremental you could have an `init.sql` to make the initial backfill slightly easier, but IMO that one-time convenience isn't worth the hassle of then having to maintain the query logic in both `init.sql` and `query.sql` for any future changes.  So ultimately I'd advise against having an `init.sql`.